bcgov/onroutebc
# ORV2-1174, ORV2-1214, ORV2-1215

# Description

Thin PR combined ORV2-1174, ORV2-1214 and ORV2-1215 to set up receipt table and persistent receipt records into the database when receipt document was generated.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Test steps:

1. Create a new application and complete the payment.
2. Go to the the database to verify if receipt record can be generated in the ORBC_RECEIPT table. 


## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-565-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Backend - DOPS](https://onroutebc-565-backend-dops.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-565-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
